rbc/tests/test_omnisci_treelite.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+from rbc.tests import omnisci_fixture
+from rbc.external import external
+
+
+@pytest.fixture(scope='module')
+def omnisci():
+    for o in omnisci_fixture(globals(), minimal_version=(5, 5), debug=not True):
+        yield o
+
+
+def test_boston_house_prices(omnisci):
+    if omnisci.compiler is None:
+        pytest.skip('test requires CLang C/C++ compiler')
+
+    device = 'cpu'
+    import numpy as np
+    import tempfile
+    try:
+        import treelite
+    except ImportError as msg:
+        pytest.skip(f'test requires treelite: {msg}')
+    try:
+        import xgboost
+    except ImportError as msg:
+        pytest.skip(f'test requires xgboost: {msg}')
```

Comment:
You can use [pytest.importorskip](https://docs.pytest.org/en/stable/reference.html#pytest-importorskip) instead of `try/except` here

Reply:
yes, good idea. Fixed.